### DISCUSSION TITLE
SO 2.0 Redirects IPv6 Request to IPv4 Address - no login possible

### DISCUSSION THREAD

**Discussion by ErwinDFIRocks** [Security-Onion-Solutions/securityonion#2119] [Unsupported Versions] [answered]
Hi All,

i am testing here the SO2.0 with IPv6.

I got stuck, if i am connectiing the machine via IPV6 or IPV6 Hostname, the SO returns the IPv4 address, no login is possible because the token is invalid.

Any idea to turn the feature off or to configure the web response to the IP or Hostname to the Request which was made from ipv6 ip or hostname? OLD SO had no issue with that behavior.

Kind Regards,
Erwin

**Comment by TOoSmOotH** [comment, marked as answer]
IPv6 is not supported right now for the management side. I have never tested using ipv6 but you need to check what your base_url is set to and switch that to your v6 address. That might get you closer but there are just new issues you will find if you get past that gate :)
